templates/solutions/telco/index.html

```diff
@@ -148,275 +159,489 @@ <h2 class="p-text--small-caps">Canonical is active in key open source initiative
+          <iframe src="https://www.youtube.com/embed/0oFnOHJrRQs?si=tVLHY2fIyBq_1B8B"
+                  class="u-embedded-media__element"
+                  title="YouTube video player"
+                  frameborder="0"
+                  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                  referrerpolicy="strict-origin-when-cross-origin"
+                  allowfullscreen></iframe>
         </div>
+        <p>See how BT and Canonical deliver 5G to UK stadiums</p>
+        <p>
+          Canonical worked with BT Group and its consumer division, EE, to drive better in-stadium connectivity across the UK as part of a larger 5G roll out strategy.
+        </p>
+        <hr class="p-rule" />
+        <p>
+          <a href="/blog/bt-group-and-canonical-deliver-5g-to-uk-stadiums ">Read the story&nbsp;&rsaquo;</a>
+        </p>
       </div>
+    </div>
+  </section>
+
+  <section>
+    <div class="row">
       <hr class="p-rule" />
-      <div class="row">
-        <div class="col-3 col-medium-2">
-          <h3 class="p-heading--5">Full security coverage</h3>
+      <div class="col-3 col-medium-6">
+        <p class="p-text--small-caps">Trusted by telco vendors around the world</p>
+      </div>
+      <div class="col-9 col-medium-6">
+        <div class="row">
+          <div class="col-3 col-medium-2">
+            <img src="https://assets.ubuntu.com/v1/dfe9e13d-bt-logo.png"
+                 style="width: 10rem;
+                        margin: -2.2rem"
+                 alt="BT Group" />
+          </div>
+          <div class="p-section--shallow col-6 col-medium-4">
+            <p>
+              <i>
+                “In the last year, we’ve moved the entire mobile base for EE onto this infrastructure. So if you’re on EE, you’re on Canonical”.
+              </i>
+            </p>
+            <hr class="p-rule" />
+            <div class="row">
+              <div class="col-3 col-medium-2">
+                <h5>Gary Roberts</h5>
+              </div>
+              <div class="col-3 col-medium-2">
+                <p class="u-text--muted">
+                  Director, Network Cloud
+                  <br />
+                  BT Group
+                </p>
+              </div>
+            </div>
+          </div>
         </div>
-        <div class="col-6 col-medium-3">
-          <p>Offload the compliance, security and support of your entire open source telco stack.</p>
-          <p>Benefit from 10 years of security maintenance with timely CVE patches and software dependency management.</p>
-          <hr class="p-rule" />
-          <p>
-            <a href="https://ubuntu.com/pro">Learn more about Ubuntu Pro&nbsp;&rsaquo;</a>
-          </p>
+        <hr class="p-rule" />
+        <div class="row">
+          <div class="col-3 col-medium-2">
+            <img src="https://assets.ubuntu.com/v1/d0bd83b2-531-5319609_nayatel-logo-png-transparent-png.png"
```

Comment:
Can you use the `image` jinja function for these images. You can get the template directly from the asset manager. This ensures proper image configuration and sends it through our CDN.

Reply:
Specifically this logo has the wrong colour background, can you reach out to Lyubo for pgn with a transparent background